modules/dataframe/src/processors/csvsource.cpp

```diff
@@ -52,23 +50,40 @@ CSVSource::CSVSource(const std::string& file)
     : Processor()
     , data_("data")
     , inputFile_("inputFile_", "CSV File", file, "dataframe")
-    , firstRowIsHeaders_("firstRowIsHeaders", "First Row Contains Column Headers", true)
-    , delimiters_("delimiters", "Delimiters", ",")
-    , doublePrecision_("doublePrecision", "Double Precision", false)
+    , firstRowIsHeaders_("firstRowIsHeaders", "First Row Contains Column Headers",
+                         CSVReader::defaultFirstRowHeader)
+    , unitsInHeaders_("unitsInHeaders", "Look for units in headers",
+                      CSVReader::defaultUnitInHeaders)
+    , unitRegexp_("unitRegexp", "Unit regexp", CSVReader::defaultUnitRegexp)
+    , delimiters_("delimiters", "Delimiters", CSVReader::defaultDelimiters)
+    , stripQuotes_("stripQuotes", "Strip surrounding quotes", CSVReader::defaultStripQuotes)
+    , doublePrecision_("doublePrecision", "Double Precision", CSVReader::defaultDoublePrecision)
+    , exampleRows_("exampleRows", "Example Rows", CSVReader::defaultNumberOfExampleRows, 0, 10000)
```

Comment:
what's the initial value? 100?

Reply:
CSVReader::defaultNumberOfExampleRows think it might still be 50, but it will read more automatically